src/libtiled/hexagonalrenderer.h

```diff
@@ -79,6 +79,10 @@ class TILEDSHARED_EXPORT HexagonalRenderer : public OrthogonalRenderer
     void drawTileLayer(QPainter *painter, const TileLayer *layer,
                        const QRectF &exposed = QRectF()) const override;
 
+    void drawTileLayer(const TileLayer *layer,
+                       const RenderTileCallback renderTileCallback,
```

Comment:
```suggestion
                       const RenderTileCallback &renderTileCallback,
```
Please change this in all cases. I don't think we want to copy the `std::function`.